sechub-doc/src/docs/asciidoc/sechub-quickstart-guide.adoc

```diff
@@ -111,30 +130,30 @@ Open the log file `./sechub-integrationtest/integrationtest-server.log` and sear
 # Everybody able to read this log file has access to these admin credentials!
 #####################################################################################
 ----
-
++
 [NOTE]
 --
 In case of an error message or failure, you can find the SecHub server log under: `./sechub-integrationtest/integrationtest-server.log`.
 
 The `{noop}` has to be excluded from the password string, therefore the password is: `int-test_superadmin-pwd`
 --
-
-These are the initial credentials when starting SecHub server in integration-test mode:
-
++
+These are the initial credentials when starting SecHub server in `integration-test` mode:
++
 SecHub Superadmin:
-[quote]
-username: `int-test_superadmin`
-password: `int-test_superadmin-pwd`
-
++
+* username: `int-test_superadmin`
+* password: `int-test_superadmin-pwd`
++
 SecHub User account:
```

Comment:
Is the line "SecHub User account" indented on purpose in the rendered version?

Reply:
Yes, this is the default SecHub User Account for the integration test mode. The integration mode uses mocked profiles and is not intended for production.

Reply:
I fixed the issue by using code blocks instead of lists.